docs/correlated_observation_scaling.rst

```diff
@@ -0,0 +1,133 @@
+Correlated Observations Scaling
+===============================
+
+Configuration
+-------------
+
+Given a configuration in the format:
+
+.. code-block:: salt
+
+    CALCULATE_KEYS:
+        keys:
+            - key: FOPR
+
+This configuration will let COS calculate a scaling factor from all data
+points in ``FOPR`` and update all data points in ``FOPR``.
+You can also specify which indices will be updated on the ``FOPR`` key:
+
+Wildcards
+^^^^^^^^^
+
+Keys can be given as wildcards:
+
+.. code-block:: salt
+
+    CALCULATE_KEYS:
+        keys:
+            - key: WOPR_OP*
+
+This will calculate a scaling factor for all keys matching ``WOPR_OP*`` where
+the asteriks will match everything. The patterns are Unix style:
+
+::
+
+    *       matches everything
+    ?       matches any single character
+    [seq]   matches any character in seq
+    [!seq]  matches any char not in seq
```

Comment:
Great that you´ve added documentation! Though this is not strictly correct, only strings with `*` will actually be evaluated by the expanding function. That being said, I´m in favor of keeping the documentation as is, and fixing the code, but that might belong in a separate issue and is outside the scope of this PR.

Reply:
Ah, I assumed it used `fnmatch`? Why just a subset?

Reply:
It uses `fnmatch`, but https://github.com/equinor/semeio/blob/9861ce8bc5780711df3809f3a028a77f7e2cef9c/semeio/jobs/correlated_observations_scaling/obs_utils.py#L39 means it might not hit `fnmatch`, which is what I meant should be changed.